src/components/ZetkinCalendar/WeekCalendar/index.tsx

```diff
@@ -80,33 +95,43 @@ const WeekCalendar = ({ orgId, baseHref, campaigns, events, focusDate, tasks }:
             date <= end;
     };
 
+    const today = new Date();
+
     return (
         <Box display="flex" flexDirection="column" height={ 1 }>
             <Box display="flex" flexDirection="column" flexGrow={ 0 } justifyContent="space-between">
                 <Box display="flex">
-                    { Array.from(Array(7).keys()).map((_, index) => (
-                        <Box
-                            key={ index }
-                            alignItems="center"
-                            data-testid="weekdays"
-                            display="flex"
-                            flexDirection="column"
-                            justifyContent="flex-start"
-                            width="100%">
-                            <Typography component="h2" data-testid={ `weekday-${index}` } variant="subtitle2">
-                                <FormattedDate
-                                    value={ new Date(new Date(calendarStartDate).setDate(calendarStartDate.getDate() + index)) }
-                                    weekday="short"
-                                />
-                            </Typography>
-                            <Typography data-testid={ `date-${index}` }>
-                                <FormattedDate
-                                    day="2-digit"
-                                    value={ new Date(new Date(calendarStartDate).setDate(calendarStartDate.getDate() + index)) }
-                                />
-                            </Typography>
-                        </Box>
-                    )) }
+                    { Array.from(Array(7).keys()).map((_, index) => {
+                        const currentDate = new Date (new Date(calendarStartDate).setDate(calendarStartDate.getDate() + index));
+                        const tomorrow = new Date (new Date(currentDate).setDate(currentDate.getDate() + 1));
+                        const isToday = isInRange(today, currentDate, tomorrow);
+                        const isPast = currentDate < today && !isToday;
+                        return (
+                            <Box
+                                key={ index }
+                                alignItems="center"
+                                className={ isPast ? classes.pastDays : '' }
+                                data-testid="weekdays"
+                                display="flex"
+                                flexDirection="column"
+                                justifyContent="flex-start"
+                                width="100%">
+                                <Typography component="h2" data-testid={ `weekday-${index}` } variant="subtitle2">
+                                    <FormattedDate
+                                        value={ new Date(new Date(calendarStartDate).setDate(calendarStartDate.getDate() + index)) }
```

Comment:
The complicated date constructor here is identical to `currentDate`, which could be used here instead.